packages/code-studio/src/dashboard/linker/LinkerOverlayContent.test.tsx

```diff
@@ -6,7 +6,7 @@ import LinkerOverlayContent from './LinkerOverlayContent';
 const LINKER_OVERLAY_MESSAGE = 'TEST_MESSAGE';
 
 function makeEventHub() {
-  const callbacks = {};
+  const callbacks = {} as Record<string, (arg: unknown) => void>;
```

Comment:
I think you can move the type before the equals. `const callbacks: Record<> = {};`. Not sure if there's any difference between the 2